src/renderers/WebGLRenderer.js

```diff
@@ -131,6 +131,29 @@ function WebGLRenderer( parameters = {} ) {
 	this.toneMapping = NoToneMapping;
 	this.toneMappingExposure = 1.0;
 
+	//
+
+	Object.defineProperties( WebGLRenderer.prototype, {
+
+		// @deprecated since r136, 0e21088102b4de7e0a0a33140620b7a3424b9e6d
+
+		gammaFactor: {
```

Comment:
Why not just do the same as with `Material.vertexTangents`?

Reply:
The getters/setters syntax of `vertexTangents` only works with ES6 classes.

Reply:
Thank you, did not notice that it is not a ES6 class...